src/screens/OtherProfile/index.js

```diff
@@ -818,7 +813,11 @@ const styles = StyleSheet.create({
     display: 'flex',
     flexDirection: 'row',
     marginBottom: 20
-  }
+  },
+  blockBtnStyle: (isAnonym) => ({
+    backgroundColor: isAnonym ? COLORS.anon_primary : COLORS.signed_primary,
+    paddingLeft: 0
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here involve adding a new style rule `blockBtnStyle` that uses the new `COLORS` object from the `theme` module. This is a good practice for maintaining consistency across the application. However, it's important to ensure that the new colors (`COLORS.anon_primary`, `COLORS.signed_primary`) are defined in the `COLORS` object and they match the intended design.


<!-- This is an auto-generated comment by OSS CodeRabbit -->